cmd/monaco/convert/command.go

```diff
@@ -19,7 +19,7 @@ package convert
 import (
 	"fmt"
 	"github.com/dynatrace/dynatrace-configuration-as-code/cmd/monaco/cmdutils"
-	"github.com/dynatrace/dynatrace-configuration-as-code/cmd/monaco/runner/completion"
```

Comment:
I wouldn't move it, It is mora a util while other packages on that level are business.

Reply:
Yes it is a util, but equally it does not belong below the runner. 

I thought about moving it to internal/completion, but we should keep all CLI-related stuff inside the cmd package

Reply:
Even when I see the content of this file, all exposed functions should go closer to caller (in same file), and unexposed reused functions are utils.

But this is some another refactoring...